net/adoptopenjdk/bumblebench/jni/ArrayReadWriteRegionBench.java

```diff
@@ -0,0 +1,31 @@
+package net.adoptopenjdk.bumblebench.jni;
+
```

Comment:
Please also add Apache v2 license header to all new Java files, see [another java file for example](https://github.com/AdoptOpenJDK/bumblebench/blob/master/net/adoptopenjdk/bumblebench/core/BumbleBench.java#L1) - applies to all new files in this PR

Reply:
Apache header is added to all new files (Java and shell scripts).